src/QuizAppGUI.java

```diff
@@ -60,7 +66,7 @@ private void startQuiz() {
 
     private void createQuiz() {
         String quizName = JOptionPane.showInputDialog(this, "Enter the topic of your quiz:", "Create New Quiz", JOptionPane.PLAIN_MESSAGE);
-        if (quizName != null && !quizName.isEmpty()) {
+        if (quizName != null && !quizName.isBlank()) {
```

Comment:
Using isEmpty checks for characters but doesn't check for spaces (i,e "   " will return true)

isBlank checks for both (i,e "   " will return false)